src/Main.java

```diff
@@ -1,5 +1,88 @@
 public class Main {
     public static void main(String[] args) {
-
+//Задача1
+        byte variableByte = 1;
+        System.out.println(variableByte);
+        short variebleShort = -200;
+        System.out.println(variebleShort);
+        int variableInt = 50000;
+        System.out.println(variableInt);
+        long variableLong = -3333333333L;
+        System.out.println(variableLong);
+        float variableFloat = 0.1234567f;
+        System.out.println(variableFloat);
+        double variableDouble = -0.123456789;
+        System.out.println(variableDouble);
+//Задача2
+        float a = 27.12f;
+        System.out.println(a);
+        long b = 987678965549L;
+        System.out.println(b);
+        float c = 2.876f;
+        System.out.println(c);
+        short d = 569;
+        System.out.println(d);
+        short e = -159;
+        System.out.println(e);
+        short f = 27897;
+        System.out.println(f);
+        byte g = 67;
+        System.out.println(g);
+//Задача3
+        byte teacherLP = 23;
+        byte teacherAS = 27;
+        byte teacherEA = 30;
+        float paperStudent = (float) 480/(teacherLP + teacherAS + teacherEA);
```

Comment:
ну тут ты получишь дробное число. т.е если логически подумать то дробного количества листов врятли бывает. т.е 6 листов и кусочек)
подумай как исправить.

Reply:
у меня в строке вывода значения переменной идет присвоение ему значения "int", сделал так на случай, если они решат с учеников деньги на бумагу собирать и нужно будет снова считать с кого сколько ;)
```suggestion
        System.out.println("There are " + (int) paperStudent + " sheets of paper for each student");   // 36 строчка
```